wikimedia/wikipedia-ios
# Image Recommendations - Empty state design review feedback

**Phabricator:** https://phabricator.wikimedia.org/T358926

### Notes
Addresses design review feedback for empty state. I was having difficulty triggering the empty state, now that we are fetching new recommendations during the flow (change made in #4802). I added a feature flag for Experimental only that bypasses some of the requirements for the entry point Explore card. This allows us to test against a Test Wiki app language, which will show the empty state due to not having any recommendations.

### Test Steps
1. On Experimental scheme, login, choose Test Wiki as primary app language and wait for Suggested Edits card to appear.
2. Tap card, confirm you see empty state.
